src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OverrideEqualsOnOverloadingOperatorEquals.Fixer.cs

```diff
@@ -27,10 +23,57 @@ public sealed override FixAllProvider GetFixAllProvider()
         }
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
```

Comment:
I considered that, but decided it against it since they might have the other rule (Equals => GetHashCode) disabled.  I don't think it would be unreasonable to file a GH issue proposing a refinement to the fixer (e.g. that we should offer both "Override Equals" and "Override Equals and GetHashCode", the latter depending on whether or not GetHashCode is already overridden).

Personally, I think that the analyzer needs refinement as well.  There are cases where shadowing makes it impossible to override Equals or GetHashCode and the only indication we can give, at present, is to decline to offer the fixes.  There should probably be separate diagnostics for "not overriding and can't".